engineering/Testing/defect-template.md

```diff
@@ -0,0 +1,93 @@
+### Defect Report template
+
+### Summary
+
+A brief summary of the bug, mostly within 60 words or below. Make sure your summary is reflecting on what the problem is and where it is.
+
+**What - Where - When (Action)**
+
+### Description:
+
+A detailed description of the bug.
+
+Use the following fields for the description field:
+
+**Reproduce steps:**
+
+Clearly, mention the steps to reproduce the bug.
+Example:
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```

Comment:
For this one, I would prefer the text like `Observed result` - it would be better

Reply:
Updated